.github/workflows/ci-cd.yml

```
@@ -0,0 +1,102 @@
+          -t ${{ secrets.DOCKER_USERNAME }}/bilbomusicbot:${{ github.ref_name }} \
+          --push .
+        
+    - name: Copy docker-compose.yml to server
+      uses: appleboy/scp-action@v0.1.7
+      with:
+        host: ${{ secrets.SERVER_HOST }}
+        username: ${{ secrets.SERVER_USER }}
+        password: ${{ secrets.SERVER_SSH_KEY }}
+        source: "docker-compose.yml"
+        target: "/opt/bilbomusicbot/"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)